contracts/cw20-staked-balance-voting/src/contract.rs

```diff
@@ -226,6 +232,36 @@ pub fn query_dao(deps: Deps) -> StdResult<Binary> {
     to_binary(&dao)
 }
 
+pub fn query_is_active(deps: Deps) -> StdResult<Binary> {
+    let threshold = ACTIVE_THRESHOLD.may_load(deps.storage)?;
+    match threshold {
+        Some(threshold) => {
+            let token_contract = TOKEN.load(deps.storage)?;
+            let staking_contract = STAKING_CONTRACT.load(deps.storage)?;
+            let total_potential_power: TokenInfoResponse = deps
+                .querier
+                .query_wasm_smart(token_contract, &cw20_base::msg::QueryMsg::TokenInfo {})?;
+            let actual_power: stake_cw20::msg::TotalStakedAtHeightResponse =
+                deps.querier.query_wasm_smart(
+                    staking_contract,
+                    &stake_cw20::msg::QueryMsg::TotalStakedAtHeight { height: None },
+                )?;
+            match threshold {
+                ActiveThreshold::AbsoluteCount { count } => to_binary(&IsActiveResponse {
+                    active: actual_power.total >= count,
+                }),
+                ActiveThreshold::Percentage { percent } => {
+                    let count = percent * total_potential_power.total_supply;
```

Comment:
This will round down. Ex: `0.5 * 7 = 3`. In the voting logic at least we take care to round up when possible. What do you think about doing that here?